### PR TITLE
[SD-3788] - Relating a story as an update

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

**Comment by akintolga** [comment]
Depends on https://github.com/superdesk/superdesk-core/pull/201

File: scripts/superdesk-archive/related-item-widget/relatedItem.js [hunk 1]
```diff
@@ -56,12 +56,45 @@
...
+                                        notify.error(gettext('Failed to associate update: ' + response.data._message));
+                                    } else {
+                                        notify.error(gettext('There is an error. Failed to associate update.'));
+                                    }
+                                });
+                    },
+                    'class': 'open',
+                    icon: 'icon-expand',
+                    condition: function(item) {
+                        var destinationRewrite = !_.contains(['published', 'corrected', 'killed', 'spiked'], $scope.item.state) &&
```
Comment (mdhaman) [inline scripts/superdesk-archive/related-item-widget/relatedItem.js]: Spiked items don't appear in the related item widget and use the `authoring.isPublished()` function.

**Comment by mdhaman** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**